manifests/params.pp

```diff
@@ -94,4 +94,5 @@
   $environment_variables      = {}
   $config_variables           = {}
   $config_kernel_variables    = {}
+  $ulimit                     = 4096
```

Comment:
@cmurphy that was a great default 15 years ago. Totally inadequate today. Everybody and their grandma changes it to be 50K+ for production machines hosting data stores, messaging brokers, proxies, etc.